tools/libs/light/libxl_pcid.c

```diff
@@ -94,7 +94,8 @@ static int process_list_assignable(libxl__gc *gc, yajl_gen gen,
     struct dirent *de;
     DIR *dir = NULL;
 
-    dir = opendir(SYSFS_PCIBACK_DRIVER);
+//    dir = opendir(SYSFS_PCIBACK_DRIVER);
```

Comment:
Do not leave commented out code.
On other hand - good commit description is required.
It is totally unclear why are you doing this change

Reply:
fixed